### DISCUSSION TITLE
Week 4 - Thoughts & Project

### DISCUSSION THREAD

**Discussion by derekscholes** [USU-ITLS-x265-23-summer/class-repo#95] [General]
# Thoughts

- **Likes**
  - I really liked the registration form. I feel like they made it fairly complex. As I looked at other examples of forms and their code, they seemed to be much more simple. I do believe the complexity of this form does help with organization and is probably better for our learning.

- **Dislikes**
  - The CSS diner I did again. I still find it somewhat difficult to understand but it was easier than the previous time I went through it. Some of the games were interesting but I have to be honest, the majority I did not like because of either difficulty or because you need to pay after a certain point. 

- **Games** (at least the ones I tried)
  - Elevator Saga
    - I somewhat liked this game. I didn't spend tons of time on it; however, I found it very difficult after adding the second elevator. I didn't attempt to code for the elevators to go to specific floors but might try that later. 
  - Screeps
    - I really disliked this game. It was very difficult and I honestly didn't understand how to play it even though I spent about 30 minutes on it. Maybe once I understand a little more about Java.
  - Code Combat
    - I did like this game; however, it only let me play a few rounds then said I needed to pay for a subscription. There is a classroom option but that requires the class to share a code, which I assume we don't have one.

# Updated Project Ideas

- **Combination of Professional profile site (CV) & Nutrition/Medical Blog**
  - A site that contains my description, past accomplishments, and portfolio of projects I have worked on.
  - Another tab with my blog (it would be another HTML page)
- **GI nutrition website**
  - Somethings that deals with GI and nutrition consults, different areas that explain what disease states I treat and prices. (not actually practicing but just an attempt of possible future work)
- **Online game (like pac man)**
  - Just a little thing to waste time and have fun.
- **Online Store**
  - Offer different items/subscriptions for sale with purchase options and checkout. Not entirely sure if I want billing... This would only be for nutrition counseling packages so it would be very simple.
 
Additional Details: I also have started some practice web pages for the CV/Blog, but I think I want to swith over to the GI nutrition website, or possibly combine all three: the blog, GI/nutrition, and CV. So far it is hard for me to decide since I want a professional website to share; however, the blog and GI/Nutrition consult site would also be beneficial for the future. Is it possible to combine the three? What are all your thoughts?

## [Mock Website](https://www.ginutritiontherapy.com): My wife gave me lots of advice on what I need to change. It actually includes three of these ideas.

**Comment by jakegros** [comment]
Hey, it looks like you have a mock website linked.  When I try to access it, it says not found.  I would love to check it out.

**Reply by derekscholes** [reply]
It should be fixed now. Feel free to give me pointers on color schemes and anythings else. [Here is the code](https://github.com/derekscholes/derekscholes.github.io) for it.

**Reply by jakegros** [reply]
Nice!  It is all workable and I love how you even have a form on there already!

**Comment by gabebott** [comment]
Your mock website looks great. I agree that the elevator game got really difficult after the second elevator. I like your project ideas as well. The online game would be fun to make and play. Great ideas on the other projects as well!

**Comment by metasean** [comment]
Your wife is clearly a wise woman!

https://duckduckgo.com/?q=color+scheme&t=newext&atb=v373-1&ia=web
